src/ObjectUtils.js

```diff
@@ -0,0 +1,87 @@
+import _ from 'underscore';
+
+/**
+ * A collection of object-related utilities.
+ */
+var ObjectUtils = {
```

Comment:
Btw better to avoid such "namespaces"—they prevent named imports and front-end [tree-shaking](https://medium.com/@Rich_Harris/tree-shaking-versus-dead-code-elimination-d3765df85c80). Instead you can just do `export function deepExtend` and `import {deepExtend}`.

Our use of namespaces largely dates from before we had a module system, when we just concatenated files together. Then, we needed the namespaces to encapsulate the implementation.